components/common/src/command/package/install.rs

```diff
@@ -178,6 +178,44 @@ impl Default for InstallMode {
     }
 }
 
+/// When querying Builder, we may not find a package that satisfies
+/// the desired package identifier, but we may have such a package
+/// already installed locally. This enum governs whether or not we
+/// should use the local package to satisfy the requirement.
+///
+/// In most cases this should be fine (and was the behavior prior to
```

Comment:
The identifier does not need to be a fully-qualified one, but yes, that's what "this" refers to.

Could you elaborate on how the wording is unclear? I'm not seeing it (but I wrote it, so I may be blind to the ambiguity).

Reply:
In the text prior to `this should be fine` we have three clauses `this` could refer to:

1. When querying Builder, we may not find a package that satisfies the desired package identifier
2. we may have such a package already installed locally.
3. This enum governs whether or not we should use the local package to satisfy the requirement.

And with the parenthetical `was the behavior`, it implies that `this` must refer to a behavior. But 1 and 2 describe situations, not behaviors and 3 describes a new behavior, not the prior one. So that's what made me confused.

If I'm understanding correctly, what `this` refers to is `if Builder doesn't find a package that satisfies the package identifier, but there is a locally installed package that does, we will use the locally installed package`. That is alluded to by the following text, but not present in the text that precedes the `this` on line 186.

Does that make sense? If not, let's jump on a zoom. I'll just be hanging out in my personal room since Slack seems to be down at the moment.

Reply:
Ah, OK, I see... I'll reword it!